modules/twinkleconfig.js

```diff
@@ -549,12 +549,28 @@ Twinkle.config.sections = [
 			enumValues: { "cat": "By categories", "alpha": "In alphabetical order" }
 		},
 		{
-			name: "customTagList",
```

Comment:
I won't be able to check this out until the new year, but won't this cause an issue with people's existing configs?

Reply:
ooohh! You're right, of course. Fixed.